docs/src/tutorials/single_stage_model.md

```diff
@@ -0,0 +1,62 @@
+# [Operation Problem with `HydroPowerSimulations.jl`](@id op_problem)
+
+**Originally Contributed by:** Rodrigo Henriquez-Auba
+
+Load packages:
+
+```@repl op_problem
+using PowerSystems
+using PowerSimulations
+using HydroPowerSimulations
+using PowerSystemCaseBuilder
+using HiGHS # solver
+```
+
+## Data
+
+!!! note
+    
+    `PowerSystemCaseBuilder.jl` is a helper library that makes it easier to reproduce examples in the documentation and tutorials. Normally you would pass your local files to create the system data instead of calling the function `build_system`.
+    For more details visit [PowerSystemCaseBuilder Documentation](https://nrel-sienna.github.io/PowerSystems.jl/stable/tutorials/powersystembuilder/)
```

Comment:
Link is failing due to PSY re-org. Either update, or preferably, set up DocumenterInterlinks.jl to PSY and use: `[PowerSystemCaseBuilder Documentation](@extref powersystembuilder)`